scripts/build_base_industry_totals.py

```diff
@@ -122,44 +110,13 @@ def create_industry_base_totals(df):
     )
     include_other = snakemake.params.other_industries
 
-    industry_list = [
-        "iron and steel",
-        "chemical and petrochemical",
-        "non-ferrous metals",
-        "non-metallic minerals",
-        "transport equipment",
-        "machinery",
-        "mining and quarrying",
-        "food and tobacco",
-        "paper, pulp and printing",
-        "paper, pulp and print",
-        "wood and wood products",
-        "textile and leather",
-        "construction",
-        "not elsewhere specified (industry)",
-        "other manuf., const. and non-fuel ind.",
-        "other manuf., const. and non-fuel min. ind.",
-    ]
-    other_list = [
-        "other manuf., const. and non-fuel ind.",
-        "other manuf., const. and non-fuel min. ind.",
-    ]
+    transaction = pd.read_csv(
+        "data/unsd_transactions.csv",
+        sep=";",
+    )
```

Comment:
Generally, it is always better to:
- load files using the helper function to avoid NAN issues especially for namibia or alike
- these data path should be loaded using inputs from snakemake

Reply:
Done